tests/detect-ldap-dn/test.rules

```diff
@@ -0,0 +1 @@
+alert tcp any any -> any any (msg:"Test ldap dn"; ldap.request.distinguished_name; content:"dc=example,dc=com"; sid:1;)
```

Comment:
can be `alert ldap` to be even more explicit

Reply:
Could you also add `startswith` and `endswith` to see that we match exactly on full buffer ?